### DISCUSSION TITLE
How to make f/F and t/T motions only work on the current line?

### DISCUSSION THREAD

**Discussion by mbenford** [folke/flash.nvim#102] [Q&A] [answered]
Is it possible to make the `char` mode highlight characters found inside the current line only?

**Comment by tsakirist** [comment]
+1

I'm also interested in this. This was a feature of `flit.nvim` that I used and I was about to open a discussion or a feature-request for this.

**Comment by folke** [comment]
What would this feature be?

Only highlight the current line, but still work multiline?

Or everything just the current line?

**Reply by tsakirist** [reply]
The whole `char` feature-mode to work only for the current line. Highlight and jump only in the line that the `char` mode was activated on.

**Comment by folke** [comment, marked as answer]
Just added this. Set `opts.modes.char.multi_line = false`

**Reply by mbenford** [reply]
Thanks! 😀

**Comment by spamwax** [comment]
would it be possible to immediately jump to the character if there is only 1 match in the current line?

This is a portion of my setup:

```lua
vim.keymap.set('o', "f", function()
  require("flash").jump({
    modes = {
      char = {
        jump_labels = false,
        multi_line = false,
      }
    }
  })
end
)
```

**Reply by tsakirist** [reply]
@spamwax This issue might be related https://github.com/folke/flash.nvim/issues/183 im also waiting for this.
